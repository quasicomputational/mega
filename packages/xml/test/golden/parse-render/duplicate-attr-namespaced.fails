<foo xmlns:a="X" xmlns:b="X" a:spam="a" b:spam="b" />
