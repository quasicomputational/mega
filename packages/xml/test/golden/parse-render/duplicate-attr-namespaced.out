Error: Duplicate attributes at (1:41-1:47) and (1:30-1:36).
